### PR TITLE
Add: Various AI/GS functions that may be useful.

### PR DESCRIPTION
**Pull request by PeterN**
Intention is to add more in separate commits.

### CODE REVIEW

**Comment by PeterN** [comment]
This would resolve #7095.

It would be useful if an AI author code try out these changes and confirm their operation and usefulness. API documentation for this branch is here: https://fuzzle.org/~petern/wip-aiapi-aidocs/ai__changelog_8hpp.html

File: src/script/api/script_company.hpp [hunk 1]
```diff
@@ -46,6 +48,52 @@ class ScriptCompany : public ScriptObject {
 		GENDER_INVALID = -1, ///< An invalid gender.
 	};
 
+	enum LiveryScheme {
+		LS_DEFAULT,
```
Comment (LordAro) [inline src/script/api/script_company.hpp]: documentation? :>

Real shame these enums have to be duplicated like this...

Comment (nielsmh) [reply src/script/api/script_company.hpp]: IMHO this PR would be good to get in for 1.9.0, any chance of this small point being fixed?

**Review by nielsmh — DISMISSED** [review]

**Review by glx22 — CHANGES_REQUESTED** [review]
Please re-run squirrel_export

**Review by glx22 — DISMISSED** [review]

**Comment by nielsmh** [comment]
I was about to merge this, but was unsure whether to squash (to hide the sausagemaking of the last two commits), to rebase as-is, or to leave it for some rebase -i.

**Review by glx22 — APPROVED** [review]

**Comment by PeterN** [comment]
Fixes #7095

**Pull request merged**